spec/lib/tabular_form_builder_spec.rb

```diff
@@ -504,15 +504,15 @@
     subject(:output) { builder.label :name }
 
     it 'should output element' do
-      expect(output).to be_html_eql %{<label class="form--label" for="user_name">Name</label>}
+      expect(output).to be_html_eql %{<label class="form--label" for="user_name" title="Name">Name</label>}
     end
 
     describe 'with existing attributes' do
-      subject(:output) { builder.label :name, 'Fear', class: 'sharknado' }
+      subject(:output) { builder.label :name, 'Fear', class: 'sharknado', title: "Fear" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.